yagpdb_docker/docker-compose.yml

```diff
@@ -21,6 +22,9 @@ services:
     volumes:
       - cert_cache:/app/cert
       - soundboard:/app/soundboard
+    ports:
+      #- '80:80' # Uncomment if you want to use external reverse proxy
+      - '443:443'
```

Comment:
Why is the one above commented but not this?

Reply:
> Why is the one above commented but not this?

It was supposed to keep people away from using http only if they have no proxy but it should be uncommented because it redirects to https if that is the case.